beta/session-1.md

```diff
@@ -0,0 +1,205 @@
+3. Run `titan install`. 
+4. Run `docker ps`. 
+>You will see two titan containers (titan-docker-server and titan-docker-launch) running. Make sure they are ‘running’ and not ‘restarting’.
+
+> For assistance, questions, or comments, please contact us via [Discord.](https://discord.gg/NVpqM7nNnT)
+
+## Step 3: Install Liquibase 4.4.3
+
+Liquibase can be downloaded from [here.]( (https://github.com/liquibase/liquibase/releases/tag/v4.4.3)). If you have previously installed Liquibase, verify that it is version 4.4.3 using `liquibase --version`.
+
+> If Liquibase is not currently on your system path, please add it.
+
+Note: The Liquibase Data extension is certified for Liquibase 4.4.3.
+
+## Step 4: Validate your Titan installation
+
+Read through [this quick start tutorial.](https://titan-data.io/getting-started)
+
+## Step 5: Download Liquibase Data extension and add to your system path.
+
+1. Download the latest [Liquibase Data extension.](https://github.com/liquibase/liquibase-data/releases) 
+2. Copy the `liquibase-data` jar file to `LIQUIBASE/lib`.
+
+> Mac will require you to go to Security & Privacy settings and allow the downloaded Liquibase Data extension to be accessed.
+
+# Create a local repository for your versioned database
+
+## Step 1: Install the PostgreSQL JDBC Driver
+
+1. Download PostgresSQL JDBC Driver from [here](https://repo1.maven.org/maven2/org/postgresql/postgresql/42.2.18/postgresql-42.2.18.jar)
+2. Copy the JAR file to your `liquibase/lib` directory.
+
+NOTE: [Liquibase Package Manager](https://github.com/liquibase/liquibase-package-manager) is an easy way to download and install Liquibase drivers and extensions for platforms like Snowflake, MongoDB, and others.
+
+## Step 2: Create a liquibase.properties
+
+Using a text editor, create a `liquibase.properties` file with the contents below.The `liquibase.properties` file allows Liquibase to communicate with your new versioned database.
+
+    driver: org.postgresql.Driver
+    url: jdbc:postgresql://localhost:5432/postgres
+    username: postgres
+    password: secret
+    changeLogFile: changelog.xml
+    repository: myrepos
+
+NOTE: The repository name can be anything you would like. For the following steps, we will use myrepos. By specifying in your liquibase.properties, you will not have to use the required argument `--respository=myrepos`. 
+
+## Step 3: Create a new data repository from a blank Docker database
+
+Think of this step as your `git init` for the versioned database. The following command will start a PostgreSQL Docker container and allow Titan to manage versions for the container. 
+
+> Your repository name can be anything you would like. We are going to use `myreps`. Feel free to change that to whatever you would like!
+
+Execute this command: 
+
+    liquibase data run --name=myrepos --env=POSTGRES_PASSWORD=secret --image=postgres
+
+NOTE: The `--name` argument should match the repository name you listed in `liquibase.properties`.
```

Comment:
I think adding the repository argument to the properties should happen after this step. I don't feel strongly about it, though.

Reply:
ok, let's ship and see if people complain. cool?